src/block/Sapling.php

```diff
@@ -66,9 +66,16 @@ public function setReady(bool $ready) : self{
 		return $this;
 	}
 
+	private function canBeSupportedBy(Block $block) : bool{
+		return
+			$block instanceof Dirt ||
+			$block instanceof Grass ||
+			$block instanceof Podzol ||
+			$block instanceof Farmland;
```

Comment:
add mycelium

Reply:
and some new dirt variants...

Reply:
It seems that mycelium can be placed from 1.18

Reply:
This should be enough for now for the blocks implemented on pmmp